packages/api-headless-cms/src/content/plugins/utils/pluralizedTypeName.ts

```diff
@@ -0,0 +1,14 @@
+import pluralize from "pluralize";
+
+// This will make it so if the content model name is a single capitalized letter
```

Comment:
I don't understand this comment.

> // Ex. A, pluralize will not capitalize the name to AS, and instead make it As

I mean, pluralize will turn `A` to `AS`, not `As`.

Did I just read it wrong?

Reply:
You are right here, I swapped the two by accident. Good catch!